packages/tocco-util/src/formField/editableValueFactory.js

```diff
@@ -69,18 +73,20 @@ const getEvents = (type, field, modelField, util) => {
   switch (type) {
     case 'single-select':
     case 'multi-select':
-      if (util.loadRelationEntity) {
+      if (util.loadRelationEntities) {
         events.onFocus = () => {
-          util.loadRelationEntity(modelField.targetEntity)
+          util.loadRelationEntities(field.id, modelField.targetEntity, {forceReload: false})
         }
       }
       break
     case 'remote':
     case 'multi-remote':
-      if (util.loadRemoteEntity) {
-        events.onFocus = () => {
-          util.loadRemoteEntity(field.id, modelField.targetEntity, '')
-        }
+      events.onFocus = () => {
+        util.loadRelationEntities(field.id, modelField.targetEntity, {
+          forceReload: true,
+          limit: 5,
```

Comment:
hardcode here? 5 or 50?

Reply:
idk. Do you have a better idea? I think either way you end up hardcoding it somewhere. We could define it in detail-form/search-form and pass it to the factory. But these would lead to 3 hardcoded settings and in my opinion in an overhead since its not really needed.

as a light alternativ i could move it to a const and move it up in the file (as kind of single point of config)